core/components/molecules/dialog/dialog.story.js

```diff
@@ -39,14 +46,21 @@ storiesOf('Dialog').add('with primary button only', () => (
 
 storiesOf('Dialog').add('with secondary button only', () => (
```

Comment:
The story is called `with secondary button only` but has 2 buttons. Create a new story please

Reply:
Done